NETProvider/src/FirebirdSql.Data.UnitTests/TestsBase.cs

```diff
@@ -109,9 +116,10 @@ public virtual void TearDown()
 				}
 			}
 			connection.Close();
+			connection.Dispose();
```

Comment:
Looks like server keeps connection for some time after it's closed from client side.